tests/density_estimator_test.py

```diff
@@ -13,14 +14,16 @@
 
 @pytest.mark.parametrize("density_estimator", (NFlowsFlow,))
 @pytest.mark.parametrize("input_dim", (1, 2))
-@pytest.mark.parametrize("context_dim", (1, 2))
-def test_api_density_estimator(density_estimator, input_dim, context_dim):
+@pytest.mark.parametrize(
+    "context_shape", ((1,), (2,), (1, 1), (2, 2), (1, 1, 1), (2, 2, 2))
+)
+def test_api_density_estimator(density_estimator, input_dim, context_shape):
     r"""Checks whether we can evaluate and sample from density estimators correctly.
 
     Args:
         density_estimator: DensityEstimator subclass.
         input_dim: Dimensionality of the input.
-        context_dim: Dimensionality of the context.
+        context_shape: Dimensionality of the context.
```

Comment:
we should be consistent with what we call `shape` and what `dim`. e.g., we could call all shapes `shape`, and all dim counts `{}_dims`